src/clients/jobManagerWrapper.ts

```diff
@@ -169,6 +169,29 @@ export class JobManagerWrapper extends JobManagerClient {
     });
   }
 
+  public async validateAndUpdateExpiration(jobId: string): Promise<void> {
+    const getJobUrl = `/jobs/${jobId}`;
+    const newExpirationDate = new Date();
+    newExpirationDate.setDate(newExpirationDate.getDate() + this.expirationDays);
+
+    const job = await this.get<JobResponse | undefined>(getJobUrl);
+    if (job !== undefined) {
+      const expirationDate = new Date(job.expirationDate as Date);
+      if (expirationDate < newExpirationDate) {
+        await this.updateJobExpirationDate(jobId);
+        this.logger.debug(`old expiration date was updated`);
```

Comment:
put the log before the update await method, should be info, please add some data - Updating (jobId, oldExpirationDate and newExpirationDate)

Reply:
done